dbpedia_Classical_musicians_by_nationality/dbpedia_Classical_musicians_by_nationality.js

```diff
@@ -0,0 +1,159 @@
+var request = require('request');
+var cheerio = require('cheerio');
+var sleep = require('system-sleep');
+var fs = require('fs');
+var url = "http://dbpedia.org/page/Category:Classical_musicians_by_nationality";
+
+
+var outputFile = "dbpedia_Classical_musicians_by_nationality.json";
+
+//helper functions to replace al occurrences of a string
+function replaceAll(str, find, replace) {
+    return str.replace(new RegExp(find, 'g'), replace);
+}
+function replaceURLAndUnderscore(str) {
+    str = str.replace("http://dbpedia.org/resource/", "")
+    return replaceAll(str, "_", " ")
+}
+//Delete outputfile if it already exists
+fs.unlink(outputFile, function () {
+    //request http://dbpedia.org/page/Category:Classical_musicians_by_nationality
+    request(url, function (error, response, body) {
+        if (error) {
+            console.log("Error: " + error);
+        }
+
+        var $ = cheerio.load(body);
+        // iterate through all of the nation_classical_composers, i.e. german_classical_composers, american_classical_composers, ...
+        $('a[rev="skos:broader"]').each(function (index) {
```

Comment:
code looks good, but I recommend reading this page: http://callbackhell.com/

Otherwise code gets unreadable!